From 74f60df680e2cd578f03f11d0f3eb046584b76bb Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Thu, 26 Jun 2014 19:28:08 +0000
Subject: [PATCH] Remove useless xcb modules

---
 CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b91f91b..483e642 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,10 +26,8 @@ set_package_properties(X11 PROPERTIES DESCRIPTION "X11 libraries"
                         PURPOSE "Required for building the X11 based workspace"
 )
 if(X11_FOUND)
-  find_package(XCB REQUIRED COMPONENTS XCB XFIXES DAMAGE COMPOSITE SHAPE SYNC RENDER RANDR KEYSYMS IMAGE SHM XTEST)
+  find_package(XCB REQUIRED COMPONENTS XCB XFIXES DAMAGE COMPOSITE SHAPE SYNC RENDER RANDR SHM XTEST)
   set_package_properties(XCB PROPERTIES TYPE REQUIRED)
-  find_package(XCB COMPONENTS ICCCM)
-  add_feature_info("XCB-ICCCM" XCB_ICCCM_FOUND "Required for building test applications for KWin")
 endif()
 
 include_directories (
-- 
1.8.3-rc3

